### PR TITLE
827: Updated header near resources tab

### PR DESCRIPTION
**Pull request by Mdemenko1**
## What (if any) features are you implementing?

Work for the issue [827](https://github.com/Gary-Community-Ventures/benefits-calculator/issues/827)

Header for the near-term resources tab

Current look: 

image

### CODE REVIEW

File: src/Components/Results/Header/Header.tsx [hunk 1]
```diff
@@ -21,8 +24,16 @@ const ProgramsHeader = () => {
 
 const NeedsHeader = () => {
   const { needs } = useResultsContext();
+  const { theme } = useContext(Context);
 
-  return <div>{needs.length}</div>;
+  return (
+    <CardContent sx={{ backgroundColor: theme.secondaryBackgroundColor }}>
```
Comment (CalebPena) [inline src/Components/Results/Header/Header.tsx:30]: Can you do this without the MUI `CardContent` component?

**Review by CalebPena — COMMENTED** [review]
How does this look on desktop?

**Comment by Mdemenko1** [comment]
> How does this look on desktop?

image

**Comment by sydney-devine** [comment]
> > How does this look on desktop?
> 
> image

Looks great - TY!

File: src/Components/Results/Header/Header.tsx [hunk 2]
```diff
@@ -50,8 +50,16 @@ const ProgramsHeader = () => {
 
 const NeedsHeader = () => {
   const { needs } = useResultsContext();
+  const { theme } = useContext(Context);
 
-  return <div>{needs.length}</div>;
+  return (
+    <div style={{ backgroundColor: theme.secondaryBackgroundColor, padding: '1rem' }}>
```
Comment (CalebPena) [inline src/Components/Results/Header/Header.tsx:56]: Can we move these styles into the CSS.

**Review by CalebPena — APPROVED** [review]
Looks good.

**Pull request merged**